src/Dto/Response/GetRes.ts

```diff
@@ -0,0 +1,29 @@
+import { Types } from "mongoose";
```

Comment:
dont use db types in request response object

Reply:
There are three different methods to obtain the output without utilizing the built-in Mongoose function.
1. By changing the ID type from objecId to string and converting each time I get an ID from the database to string this whould be a problem when geting lots of data at once
2. or by setting the type to any
3. or by using the external package for this perticular reason and use theres inbuilt type

Reply:
what should i do

Reply:
loggers should have context ex: some identifer from request (adding phone number for name=) (updating contact for id=request.id)
For the above comment, I have used an external package named uuid to generate the unique ID, and I am manually printing it for each log. Should I change that and make it a template inside Winston? If I do that, then I need to put a function inside the Winston config file to set that template, which will be receiving the req and res from the app.ts file.

Reply:
what should i do

Reply:
conversion needs to be done to string and string to objectID when getting request
also uuid wont solve the problem we need to log some info from request 
if not we need to return uuid in response header

Reply:
I will be logging the name and ID of the data.